style-dictionary/classic/colors.ts

```diff
@@ -104,6 +105,7 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorTextExpandableSectionDefault: '{colorTextInteractiveDefault}',
   colorTextExpandableSectionHover: '{colorTextInteractiveHover}',
   colorTextExpandableSectionNavigationIconDefault: '{colorTextIconCaret}',
+  colorTextHeadingPrimary: { light: '{colorGrey900}', dark: '{colorGrey200}' },
```

Comment:
We also need to add `colorTextHeadingDefault:  { light: '{colorGrey900}', dark: '{colorGrey200}' },` if we don't want the changes to apply to classic as well (which I don't think we do).